common/src/main/kotlin/flank/common/Files.kt

```diff
@@ -14,6 +15,24 @@ val userHome: String by lazy {
 
 fun String.deleteFile() = Paths.get(this).delete()
 
+fun linkFiles(
+    link: String,
+    target: String
+) = if (isWindows) copyDirectory(target, link) else createSymbolicLink(link, target)
+
+fun copyDirectory(sourceDirectoryLocation: String, destinationDirectoryLocation: String) {
+    Files.walk(Paths.get(sourceDirectoryLocation))
+        .forEach { source: Path ->
+            val destination =
+                Paths.get(destinationDirectoryLocation, source.toString().substring(sourceDirectoryLocation.length))
+            try {
+                Files.copy(source, destination)
+            } catch (e: IOException) {
+                logLn(e.localizedMessage)
+            }
+        }
+}
+
 fun createSymbolicLink(
     link: String,
     target: String
```

Comment:
Should not we apply similar logic for this method?

Reply:
We can but then its no longer creating a SymbolicLink and the name should change.

Reply:
Maybe this function signature should have a `typealias` so we could switch reference to function for specific linking strategy depending on OS type.

Reply:
As mentioned we are copying for windows and symlinking for everyone else. I dont think there should be a typealias as its fairly self explanatory.